### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by VasiliZas**
Task about Numbers

### CODE REVIEW

File: hw2/NumberService.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+public class NumberService {
+    int start;
+    int finish;
+    public static int result;
+    public static int i;
+
+    public int rangeSum(int start, int finish) {
+        if (start > finish) {
+            while (start >= finish) {
+                result = start + result;
```
Comment (javaGuruBY) [inline hw2/NumberService.java:12]: At this point if result already has value, it'll be used. Should you reset it?

File: hw2/NumberService.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                start = start - 1;
```
Comment (javaGuruBY) [inline hw2/NumberService.java:13]: encapsulate this into single mehtod

File: hw2/NumberService.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package jis8.java.homework.hw2;
+
+public class NumberService {
+    int start;
+    int finish;
+    public static int result;
+    public static int i;
+
+    public int rangeSum(int start, int finish) {
+        if (start > finish) {
```
Comment (javaGuruBY) [inline hw2/NumberService.java:10]: if () else () encapsulate to another single method.

File: hw2/NumberService.java (hunk 1 above)
Comment (javaGuruBY) [reply hw2/NumberService.java:12]: result =+ start

File: hw2/NumberService.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
+package jis8.java.homework.hw2;
+
+public class NumberService {
+    int start;
+    int finish;
+    public static int result;
```
Comment (javaGuruBY) [inline hw2/NumberService.java:6]: this variable is redundunt

File: hw2/NumberService.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        } else {
+            while (start <= finish) {
+                result = start + result;
+                start = start + 1;
+            }
+        }
+        return result;
+    }
+
+    public int rangeEvenCount(int start, int finish) {
```
Comment (javaGuruBY) [inline hw2/NumberService.java:24]: all the same as for result and method above

File: hw2/NumberServiceTest.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
+package jis8.java.homework.hw2;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (javaGuruBY) [inline hw2/NumberServiceTest.java:5]: Use Ctrl+Alt+O to optimize imports

File: hw2/NumberServiceTest.java [hunk 7]
```diff
@@ -0,0 +1,22 @@
+package jis8.java.homework.hw2;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NumberServiceTest {
+int exp1;
+int exp2;
```
Comment (javaGuruBY) [inline hw2/NumberServiceTest.java:9]: not used, delete it.

File: hw2/NumberServiceTest.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+    public void rangeSum() {
+    assertEquals(25,numberService.rangeSum(3,7));
+
+    }
+
+    @Test
+    public void rangeEvenCount() {
+        assertEquals(4,numberService.rangeEvenCount(9,2));
+    }
+}
```
Comment (javaGuruBY) [inline hw2/NumberServiceTest.java:22]: add new line in the end of the file

File: hw2/NumberServiceTest.java [hunk 9]
```diff
@@ -0,0 +1,22 @@
+package jis8.java.homework.hw2;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NumberServiceTest {
+int exp1;
+int exp2; 
+NumberService numberService = new NumberService();
```
Comment (javaGuruBY) [inline hw2/NumberServiceTest.java:10]:
```
NumberService numberService;

@Before
public void setUp() {
    numberService = new NumberService();
}
```

File: hw2/NumberServiceTest.java (hunk 8 above)
Comment (lamaconda) [inline hw2/NumberServiceTest.java:22]: В конце должна оставаться пустая строчка

File: hw2/NumberService.java [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+                    i++;
+                }
+                start = start + 1;
+
+            }
+
+        }
+        return i;
+    }
+}
```
Comment (lamaconda) [inline hw2/NumberService.java:44]: В конце пустая строчка

**Review by javaGuruBY — CHANGES_REQUESTED** [review]
PTAL, and please use maven quickstart project and sonar scanner.

**Review by lamaconda — CHANGES_REQUESTED** [review]
add empty line in the end of programm

**Review by lamaconda — APPROVED** [review]
lgfm, but you should add new line in the end of the file

**Pull request closed without merging**